public_html/behavioralhealth/estacada/resources.html

```diff
@@ -130,345 +130,578 @@ <h1 class="blue no_margin some_margin" style="font-size: 40px;">Behavioral Healt
                             <hr class="blue-hr">
 
                             <div class="pad_bottom">
-                                <h4 class="blue no_margin some_margin">EMERGENCY WARMING SHELTER - 971-235-5678</h4>
-                                <p>239 NE Main St., Estacada, OR 97023<br>
-                                    Open from 6pm - 8am. Food provided.
-                                </p>
-                            </div>
-                            <div class="pad_bottom">
-                                <h4 class="blue no_margin some_margin">CLACKAMAS COUNTY COORDINATED HOUSING ACCESS (CHA) - 503-655-8575</h4>
-                                <p><a href="https://www.clackamas.us/communitydevelopment/cccha.html" target="_blank">www.clackamas.us/communitydevelopment/cccha.html</a><br>
-                                    For individuals and families experiencing homelessness or a housing crisis in Clackamas County.
+                                <h4 class="blue no_margin some_margin">BEAVERCREEK HEALTH CLINIC</h4>
+                                <p> Phone: 503-655-8471<br>
+                                    110 Beavercreek Road, Suite 100 Oregon City, OR 97045<br>
+                                    Website: <a target="_blank" href="https://www.clackamas.us/healthcenters/beavercreek.html#dentalservices">https://www.clackamas.us/healthcenters/beavercreek.html#dentalservices</a><br>
+                                    Accepts OHP, uninsured based upon income<br>
+                                    Emergency dental also<br>
+                                    Another location: Sunnyside Health Center: 9775 SE Sunnyside Road, Suite 200, Clackamas, OR 97015
                                 </p>
                             </div>
                             <div class="pad_bottom">
-                                <h4 class="blue no_margin some_margin">HUMAN SOLUTIONS - 503-477-9724 (24/7 shelter)</h4>
-                                <p>503-548-0200 (number for affordable housing)<br>
-                                    16015 SE Stark St., Portland, OR 97233 (shelter location)<br>
-                                    Or visit <a href="https://www.humansolutions.org" target="_blank">www.humansolutions.org</a><br>
-                                    Low income housing in multiple locations and 24/7 shelter for homeless families with children and pregnant women.
+                                <h4 class="blue no_margin some_margin">NEIGHBORHOOD HEALTH CENTER</h4>
+                                <p> Phone: (503) 941-3064<br>
+                                    19029 S Beavercreek Rd., Oregon City, OR 97045<br>
+                                    Website: <a target="_blank" href="https://www.nhcoregon.org/locations/">https://www.nhcoregon.org/locations/</a><br>
+                                    Other locations: 728 Molalla Ave., Oregon City, OR 97045<br>
+                                    Phone: (503) 656-9030
```

Comment:
phone number listed twice. remove this second one

Reply:
The were two phone numbers listed in the word doc because of two different locations. I just made it more clear that second phone number was associated with the second location.